TiFBackendUtils/SQLExecutable/index.ts

```diff
@@ -1,5 +1,39 @@
-import { planetscaleConnection } from "../Planetscale/utils.js"
+import dotenv from "dotenv"
+import mysql from "mysql2/promise.js"
+import { z } from "zod"
 import { SQLExecutable } from "./utils.js"
 export { SQLExecutable }
 
-export const conn = new SQLExecutable(planetscaleConnection)
+dotenv.config()
+
+const EnvVarsSchema = z
+  .object({
+    DATABASE_HOST: z.string(),
+    DATABASE_PASSWORD: z.string(),
+    DATABASE_USERNAME: z.string(),
+    DATABASE_NAME: z.string()
+  })
+  .passthrough()
+
+const envVars = EnvVarsSchema.parse(process.env)
```

Comment:
This should be in a separate file since we'll for sure need env vars that are not related to the DB.

Reply:
True, I'll update after more .env vars are added